locale/en/blog/weekly-updates/weekly-update.2015-05-15.md

```diff
@@ -19,11 +19,11 @@ If you would like to see the convergence plan in detail, please check it at [jas
 ## Community Updates
 
 * [Performance Showdown](https://raygun.io/blog/2015/05/performance-showdown-node-js-vs-io-js-v2-0-0/): Node.js vs. io.js v2.0.0.
-* Microsoft has [node.js](https://github.com/Microsoft/node) running on their JavaScript VM `Chakra`.
+* Microsoft has [Node.js](https://github.com/Microsoft/node) running on their JavaScript VM `Chakra`.
 * {node,io}.js [benchmark graphs](http://thlorenz.com/benchgraph/).
```

Comment:
```suggestion
* {Node,io}.js [benchmark graphs](http://thlorenz.com/benchgraph/).
```

Reply:
It's the name of the linked website so should probably stay as-is.